platforms/qurt/src/px4_layer/px4_qurt_impl.cpp

```diff
@@ -166,3 +166,9 @@ int fputc(int c, FILE *stream)
 {
 	return c;
 }
+
+int putchar(int character)
+{
+	PX4_ERR("Error: Calling unresolved symbol stub:[%s]", __FUNCTION__);
```

Comment:
@dagar I just talked to @bkueng about this and he had some concerns about this implementation.
Specifically how can we be sure that PX4_ERR (which uses printf) does not end up calling putchar again?

Reply:
I agree, it's a bit questionable. I don't have a full understanding of how this symbol resolution is expected to work qurt side.